README.md

```diff
@@ -5,11 +5,56 @@
+
+### Como estamos tentando atingir esses objetivos?
+Algumas coisas que estamos fazendo atualmente são:
+- Apresentar anualmente o GEMP e o "mundo" de programação competitiva na semana dos calouros da UFPA.
+- Treinar para continuar melhorando nossas participações em finais brasileiras da Maratona de Programação.
+- Ensinar os tópicos de computação para novos alunos por meio de encontros semanais e discussões em grupo.
+- Tentar firmar parcerias com a faculdade de computação ou laboratórios da universidade.
+
+Além disso, incentivamos um **ritmo saudável de estudo**, além de dar suporte levando em conta as especificidades de cada pessoa. 
+
+Em relação aos estudos, **nosso foco é menos na quantidade e mais na qualidade desse treino**. Acreditamos que praticar de forma **consistente** e com os **melhores materiais disponíveis** vale mais do que horas a fio pelas as madrugadas. E por isso os membros do GEMP também tem a preocupação de fazer **curadoria** sobre as principais referências de estudo em programação competitiva.
+
+### Nossos valores
+Para construir uma *boa* comunidade em volta desta cultura, não bastam apenas números. Estamos sempre tentando nos desenvolver mais como pessoas de forma que isso também tenha um impacto real na interação com nossos colegas e no processo de aprendizagem coletivo. Efetivamente estamos sempre trabalhando:
+
+- **Responsabilidade**. Quando vamos explicar ou ensinar alguma coisa sempre tomamos a responsabilidade, dado que se a outra pessoa não entende completamente a explicação, o problema não é dela por não entender, e sim nossa por não explicar da melhor forma.
+
+- **Respeito**. Somos contra qualquer tipo de discriminação, e mais ainda, somos contra qualquer tipo de atitude com objetivo de inferiorizar o outro, então nenhuma pergunta é "idiota" e respeitamos o ritmo de aprendizado de cada pessoa.
+
+- **Humildade**. Ninguém é perfeito, e estamos sempre melhorando, então estamos sempre abertos a críticas construtivas.
+
+### Sobre o processo de aprendizagem
+
+A primeira coisa que estaremos desenvolvendo é a nossa **autonomia** no processo de aprendizagem. Isso **não significa fazer tudo sozinho**, e sim que estaremos ativamente sempre indo atrás daquilo que queremos aprender, pesquisando documentação, pedindo recomendação de boas referências, e, claro, pedindo ajuda quando necessário.
+
+Alguém disse que **"Se você quer aprender algo, dê aula sobre aquilo"** e, bem, nós levamos isso no coração. Encaramos as oportunidades de ensinar algo para os novos membros como formas de solidificar os conhecimentos que adquirimos até então, pois não basta que faça sentido apenas para nós, mas que consigamos explicar de forma que faça sentido para o outro também.
```

Comment:
Sobre "Se você quer aprender algo, dê aula sobre aquilo":
“If you want to master something, teach it. A great way to learn is to teach.” Richard Feynman – theoretical physicist

https://www.richmondshare.com.br/if-you-want-to-master-something-teach-it/

Reply:
Boooooa! Tava na dúvida se era dele, até procurei mas não tinha achado ainda auhsuahus. Vou colocar a menção

Reply:
Adicionado em [43bc97f](https://github.com/Rockett12/GEMP-UFPA/pull/21/commits/43bc97f07caadea920700482add64c79dd0e6f9c)